llvm/test/MC/AArch64/directive-arch-section-alignment.s

```diff
@@ -0,0 +1,21 @@
+// RUN: llvm-mc -triple aarch64-windows -filetype obj -o %t.obj %s
+// RUN: llvm-objdump -d -r %t.obj | FileCheck %s
+
+.section sec00, "ax"
+nop
+nop
+nop
+.section sec01, "ax"
+.balign 4
+nop
+nop
+nop
+
+// CHECK: 0000000000000000 <sec00>:
+// CHECK-NEXT: 0: d503201f      nop
+// CHECK-NEXT: 4: d503201f      nop
+// CHECK-NEXT: 8: d503201f      nop
+// CHECK: 0000000000000000 <sec01>:
+// CHECK-NEXT: 0: d503201f      nop
```

Comment:
How are we checking the alignment here to ensure it has been set to 4, especially in the `sec00` section?

Reply:
I am not sure how to check that. I thought if the address is 4 bytes increment, that will imply an alignment of 4? But I don't know yet how to do it better because since I've changed the code, it doesn't emit `p2align` directive anymore

Reply:
If you add the `-h` option to the `llvm-objdump` that should give you the section header information, which should include the alignment.

Reply:
I have redone the test and now I am checking for the section alignment with both `readobj` and 'readelf'